### PR TITLE
fix: improved the algorithm

### PR DESCRIPTION
**Pull request by amitojsingh**
To Test Please create a new user. 

Tasks Completed. 
1. Next option redirects to infomation page for wrong answer. 
2. Fix the information page design 
3. improve algorithm

### CODE REVIEW

File: src/pages/dashboard/hooks/useNew.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import { GameScreen, WordShabadavaliDB } from 'types/shabadavalidb';
+import { getRandomWord } from 'database/default';
+import { getQuestionsByWordID } from 'database/default';
+import {  WordType } from 'types';
+import { createGameScreen } from '../utils';
+import ALL_CONSTANT from 'constants/constant';
+const useNew = ()=>{
+  const addWordIfNotExists = (word: WordType, learningWords:WordShabadavaliDB[]) => {
```
Comment (MySmilingTurban) [inline src/pages/dashboard/hooks/useNew.ts:8]: add empty between imports and functions
add adequate spaces for better formatting

File: src/pages/dashboard/hooks/useQuestions.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import {  User, GameScreen, WordShabadavaliDB } from 'types/shabadavalidb';
+import { getWordsFromUser } from 'database/shabadavalidb';
+import { createGameScreen } from '../utils';
+import ALL_CONSTANT from 'constants/constant';
+import { getQuestionsByWordID } from 'database/default';
+import { QuestionData } from 'types';
+
+const useQuestions = (user:User)=>{
+
```
Comment (MySmilingTurban) [inline src/pages/dashboard/hooks/useQuestions.ts:9]: nit: remove empty line

**Review by MySmilingTurban — CHANGES_REQUESTED** [review]

File: src/pages/dashboard/hooks/useNew.ts (hunk 1 above)
Comment (amitojsingh) [reply src/pages/dashboard/hooks/useNew.ts:9]: fixed

File: src/pages/dashboard/hooks/useQuestions.ts (hunk 2 above)
Comment (amitojsingh) [reply src/pages/dashboard/hooks/useQuestions.ts]: fixed

File: src/pages/dashboard/hooks/useNew.ts (hunk 1 above)
Comment (MySmilingTurban) [reply src/pages/dashboard/hooks/useNew.ts:9]: ()=>{ should be () => {

**Review by MySmilingTurban — CHANGES_REQUESTED** [review]
also, resolve conflicts

File: src/pages/dashboard/hooks/useNew.ts (hunk 1 above)
Comment (amitojsingh) [reply src/pages/dashboard/hooks/useNew.ts]: fixed

File: src/pages/word/layout.tsx
Comment (MySmilingTurban) [inline src/pages/word/layout.tsx:1]: @amitojsingh veerji, this was intentionally added only for word pages to allow users to navigate back and forth through word information.

**Review by MySmilingTurban — APPROVED** [review]
looks good to me!
left a comment about removal of back button in words layout page.

**Comment by MySmilingTurban** [comment]
@amitojsingh , should we push this to dev?

**Comment by amitojsingh** [comment]
Nopes, Veer Ji I tested this. This is also stuck. Because I merge dev branch here. 
We need to first fix the dev branch and then we can merge this.

**Review by MySmilingTurban — APPROVED** [review]
Looks good to me!

**Pull request merged**